covid_xprize/validation/scenario_generator.py

```diff
@@ -165,6 +164,13 @@ def generate_scenario(start_date_str, end_date_str, raw_df, countries=None, scen
     return ips_df
 
 
+def phase1_update(latest_df):
+    # Feb 2, 2021: Handle US Virgin Islands: was a region of 'United States' for phase 1, but is now a country
```

Comment:
Particular handling of US Virgin Island to keep the dataframe compliant with Phase 1 countries and regions.